create-update.sh

```diff
@@ -130,6 +133,10 @@ if [ $NOPUBLISH -eq 0 ]; then
 	sudo cp "$PWD/.mix-version" "$STATE_DIR/www/version/format$FORMAT/latest"
 fi
 
-# step 6: archive the swupd-server logs for this mix build
+# step 6: create delta packs and super packs
```

Comment:
These two scripts (step 6 overall) should be left out of create-update.sh and run as a separate part of update, since delta packs are not required to create a fully working update.